app/serializers/api/v2/certificates/certificates_serializer_list.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module V2
+    module Certificates
+      class CertificatesSerializerList
```

Comment:
Probably I'd rename the other CertificateSerializer to CertificateSearchSerizalizer and steal that. If this is more useful (and I think it probably is) we'll probably want to use this in the future

Reply:
I agree with the renaming (certificates_list_serializer and certificates_search_serializer),
about the `stealing` do you mean, reusing it?

In this case some duplication is useful to avoid ugly conditional attributes (in the serializers and where those are called)

Reply:
Yeah, so this becomes CertificateSerializer